python/pyspark/sql/decorators.py

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql.types import StringType
+from pyspark.sql.functions import udf as _udf
+
+
+def udf(returnType=StringType()):
```

Comment:
Of course it could be just moved to `sql.functions` and renamed explicitly as `udf_decorator`. Then user can use an alias to get shorter call:

    from sql.functions import udf_decorator as udf